types/react-redux/react-redux-tests.tsx

```diff
@@ -10,6 +10,265 @@ import objectAssign = require('object-assign');
 // https://github.com/rackt/react-redux/blob/master/docs/quick-start.md#quick-start
 //
 
+// Test cases written in a way to isolate types and variables and verify the
+// output of `connect` to make sure the signature is what is expected
+
+namespace Empty {
```

Comment:
namespacing allows all type and variable definitions to be isolated preventing a lot of numbers from being appended to types and variables making the test easier to understand and isolated. It was a bit odd that if I forgot to defined `mapStateToProps` that I'd get a type error about not having `{ value: number }` since that is the signature of a globally defined function